package.json

```diff
@@ -39,6 +39,7 @@
     "plugins": []
   },
   "dependencies": {
+    "@tgrospic/rnode-client-js": "file:.yalc/@tgrospic/rnode-client-js",
```

Comment:
why is this here? The current design is to use a submodule; does that not suffice?

Does it work? Hm... netlify is not testing this PR because it targets the `progress-tour` branch. I wonder if it's a good idea to target that branch. I'm a little swapped out.

Reply:
that did work but we just took out the .yalc reference which also worked.
    "rnode-client-js": "@tgrospic/rnode-client-js",